Libraries/LibWeb/DOM/Document.h

```diff
@@ -877,6 +876,11 @@ class Document
 
     GC::Ptr<Layout::Viewport> m_layout_root;
 
+    GC::Ptr<Node> m_hovered_node;
+    GC::Ptr<Node> m_inspected_node;
+    GC::Ptr<Node> m_highlighted_node;
+    Optional<CSS::Selector::PseudoElement::Type> m_highlighted_pseudo_element;
```

Comment:
It looks like we no longer track which pseudo-element is being inspected? Though, everything still works fine when I test it so I'm misunderstanding something...

Also makes me notice we don't track that for hovering. We could probably use a nice type for "DOM Node possibly with an associated pseudo-element" at some point.

Reply:
Right, we don't need it for the inspected node, only the highlighted node (no idea about the hovered node though). The only user of the cached inspected node is the `$0` getter.

Reply:
Ah, that explains it!

$0: i cri evertiem 😢